<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ServerListDlg</class>
 <widget class="QDialog" name="ServerListDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Server List</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../images/icons.qrc" >:/default/server.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="networkTree" >
       <property name="dragEnabled" >
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="addButton" >
         <property name="text" >
          <string>&amp;Add...</string>
         </property>
         <property name="icon" >
          <iconset resource="../images/icons.qrc" >:/default/edit_add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&amp;Edit...</string>
         </property>
         <property name="icon" >
          <iconset resource="../images/icons.qrc" >:/default/edit.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>&amp;Delete</string>
         </property>
         <property name="icon" >
          <iconset resource="../images/icons.qrc" >:/default/edit_remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>85</width>
           <height>296</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="showOnStartup" >
       <property name="text" >
        <string>&amp;Show this dialog on startup</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>291</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Connect</string>
       </property>
       <property name="icon" >
        <iconset resource="../images/icons.qrc" >:/default/connect_creating.png</iconset>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>C&amp;lose</string>
       </property>
       <property name="icon" >
        <iconset resource="../images/icons.qrc" >:/default/button_cancel.png</iconset>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="widget" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>739</width>
     <height>393</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>408</y>
     <width>739</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" >
   <property name="geometry" >
    <rect>
     <x>660</x>
     <y>10</y>
     <width>87</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../images/icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>ServerListDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ServerListDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
